frontend/src/app/app.routes.ts

```diff
@@ -153,6 +154,10 @@ export const routes: Routes = [
             }
           }
         ]
+      },
+      {
+        path: 'cities',
+        component: CitiesUploadComponent
```

Comment:
Yes that's what I meant: maybe we should not even display the admin pages to the none admin users?